harfbuzz-sys/harfbuzz/src/Makefile.am

```diff
@@ -308,7 +308,7 @@ EXTRA_DIST += \
 	$(NULL)
 # We decided to add ragel-generated files to git...
 #MAINTAINERCLEANFILES += $(RAGEL_GENERATED)
-$(srcdir)/%.hh: $(srcdir)/%.rl
+$(srcdir)/%.hh:
```

Comment:
If we apply this change, we need some way to preserve this when we update harfbuzz to new releases, which happens fairly regularly.